homeassistant/components/homematicip_cloud/binary_sensor.py

```diff
@@ -218,6 +218,19 @@ def available(self):
         # the individual availability of group members.
         return True
 
+    @property
+    def icon(self):
+        """Return the icon to use in the frontend, if any."""
+        if super().icon:
+            return super().icon
+        if self._device.motionDetected or self._device.presenceDetected:
+            return 'mdi:motion-sensor'
```

Comment:
We should use device class instead of custom icons.

Reply:
I agree.